src/shared/components/plots/MultipleCategoryBarPlot.tsx

```diff
@@ -917,3 +929,47 @@ export default class MultipleCategoryBarPlot extends React.Component<
         );
     }
 }
+
+type QPLabelProps = {
```

Comment:
Could we change this to `PQLabelProps`, just wanted to make this same as the name you give for `PQValueLabel`

Reply:
Solved in [846942a](https://github.com/cBioPortal/cbioportal-frontend/pull/4234/commits/846942aa16d04fc787ad358d5004dfd3e43c7828)